src/Illuminate/Config/Repository.php

```diff
@@ -18,15 +19,40 @@ class Repository implements ArrayAccess, ConfigContract
      */
     protected $items = [];
 
+    /**
+     * Configs loaded from cache.
+     *
+     * @var bool
+     */
+    private bool $loadedFromCache = false;
+
+    /**
+     * Path of configuration files.
+     *
+     * @var string
+     */
+    private string $configPath;
+
     /**
      * Create a new configuration repository.
      *
      * @param  array  $items
-     * @return void
+     * @param  string  $configPath
+     *
+     * @throws Exception
      */
-    public function __construct(array $items = [])
+    public function __construct(array $items = [], string $configPath = 'test')
```

Comment:
This is a breaking change and would require targeting Laravel 11 (master branch)

Reply:
This change is not considered a breaking change, and it could be integrated into Laravel 10 without causing fundamental issues. More substantial changes are feasible for Laravel 11 if needed.